lex-examples/gemini.py

```
@@ -0,0 +1,145 @@
+    results = words_to_add.get(word, [])
+    return results
+
+# دالة جلب الأشكال من Wikidata
+
+
+def get_forms_from_lexeme(lexeme_id):
+    url = f"https://www.wikidata.org/wiki/Special:EntityData/{lexeme_id}.json"
+    headers = {
+        "User-Agent": "MyQuranLexemeBot/1.0 (example@example.com)"
+    }
+    r = requests.get(url, headers=headers)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)